src/khoj/processor/content/notion/notion_to_entries.py

```diff
@@ -112,7 +112,9 @@ def process(
                             page_entries = self.process_page(p_or_d)
                             current_entries.extend(page_entries)
 
-        return self.update_entries_with_ids(current_entries, user)
+        current_entries = TextToEntries.split_entries_by_max_tokens(current_entries, max_tokens=256)
+
+        return self.update_entries_with_ids(current_entries, user=user)
```

Comment:
Oh, was the issue just that we weren't fixing the entries by max token? Why was that breaking indexing of Notion entries?

Reply:
Ah, that method also generally handles logic for splitting up the data into chunks and creating the `Entry` items that make it easy to write to the db.